src/Reflection/ReflectionProperty.php

```diff
@@ -452,6 +455,7 @@ public function getValue(object|null $object = null): mixed
 
             $closure = Closure::bind(fn (string $implementingClassName, string $propertyName): mixed => $implementingClassName::${$propertyName}, null, $implementingClassName);
 
+            /** @phpstan-ignore-next-line function.alreadyNarrowedType instanceof.alwaysTrue */
```

Comment:
Are the stubs wrong in phpstan upstream? Or in psalm? 🤔

Reply:
@ondrejmirtes Can you check please?

Reply:
```
/** @phpstan-ignore-next-line function.alreadyNarrowedType instanceof.alwaysTrue */
```

This ignores everything on the next line, not just those identifiers - you need `@phpstan-ignore function.alreadyNarrowedType instanceof.alwaysTrue`.

Reply:
Alright so PHPStan is complaining that `assert($closure instanceof Closure);`. `$closure` is what gets returned by `Closure::bind`. When it can return `null`?

Reply:
This is according to upstream types: https://www.php.net/manual/en/closure.bind.php

Reply:
Alright, I'm going to return a benevolent union there. It's not going to complain in your case but it's not going to require handling the null either.

Reply:
@kukulich BTW, happy to iterate over this in future changes: should we move forward with this patch as-is, meanwhile?

Reply:
@Ocramius Yes, we can :)